ebms-send-in/src/main/kotlin/no/nav/emottak/fellesformat/FellesFormatWrapper.kt

```diff
@@ -32,7 +32,7 @@ private fun createFellesFormatMottakEnhetBlokk(sendInRequest: SendInRequest): EI
         herIdentifikator = sendInRequest.addressing.from.partyId.getIdentifikatorByType("HER")
         orgNummer = sendInRequest.addressing.from.partyId.getIdentifikatorByType("orgnummer", "ENH")
         meldingsType = "xml"
-        partnerReferanse = sendInRequest.cpaId
+        partnerReferanse = sendInRequest.partnerId?.toString() ?: ""
```

Comment:
Frikort vil per nå fortsatt ha `cpaId` i dette feltet, så inntil videre må vi videreføre dette for frikort-spørringene (HarBorgerFrikort, HarBorgerEgenandelFritak, HarBorgerFrikortMengde).